### PR TITLE
Return empty result (instead of trying to iterate on None object) whe…

### PR DESCRIPTION
**Pull request by tomplast**
…n search returns no results.

### CODE REVIEW

File: resources/lib/api/graphql.py [hunk 1]
```diff
@@ -170,6 +170,10 @@ def getSearchResults(self, query_string):
     if not json_data:
       return None
     results = []
+
+    if not json_data['searchPage']['flat']:
```
Comment (linqcan) [inline resources/lib/api/graphql.py:173]: Please add this before L172 as `results` doesn't need to be initalized yet.

**Comment by linqcan** [comment]
Thanks for updating. Before I can merge this, please squash your changes to one commit, please.

**Comment by tomplast** [comment]
@linqcan I have squashed the commits into one now, I hope it's okay.

**Comment by linqcan** [comment]
Excellent, thank you for your contribution!

**Pull request merged**